ci/playbooks/deploy_podified_ctlplane.yaml

```diff
@@ -0,0 +1,81 @@
+---
+- hosts: controller
+  tasks:
+    - name: Install Ansible, Make, Tar
+      become: true
+      ansible.builtin.package:
+        name:
+          - ansible-core
+          - make
+          - tar
+        state: latest
+
+    - name: Install devsetup dependencies
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls/devsetup"
+        target: download_tools
+
+    - name: Create OpenStack namespace
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls"
+        target: namespace
+        params: &install_yamls_out
+          OUT: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls/out"
+
+    - name: Generate olm artifacts
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls"
+        target: openstack_prep
+        params: *install_yamls_out
+
+    - name: Deploy Openstack Operators
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls"
+        target: openstack
+        params: *install_yamls_out
+
+    - name: Wait for operator to get deployed
+      ansible.builtin.shell: |
+        for retry in {1..10}; do
+          if ! (oc -n openstack get pods --no-headers=true | grep -viqE 'completed|running'); then
+            break;
+          else
+            echo "Waiting pods to be ready";
+            sleep 10 ;
+          fi ;
+        done
+        oc get pods;
+        oc get csv;
+        INSTALLED_CSV=$(oc get subscription openstack-operator -o jsonpath='{.status.installedCSV}')
+        oc get csv ${INSTALLED_CSV} -o 'jsonpath={.spec.install.spec.deployments[*].name}' | \
+          timeout 240 xargs -I '{}' -d ' ' sh -c 'oc wait --for=condition=Available deployment {} --timeout=-1s'
+
+    - name: Generate OpenStack CR
```

Comment:
again this is not needed we can just call
openstack_deploy

Reply:
Done